packages/sources/amberdata/test/unit/price.test.ts

```diff
@@ -20,6 +20,10 @@ describe('price endpoint', () => {
         name: 'quote not supplied',
         testData: { id: jobID, data: { base: 'LINK' } },
       },
+      // {
+      //   name: 'overrides test',
+      //   testData: { id: jobID, data: { from: 'DIGG', to: 'BTC' } },
+      // },
```

Comment:
Is this something we should uncomment or remove?

Reply:
We can delete them, I was just using them for convenience. I'll delete them in the next commit